pkg/cmd/pulumi/operations/io.go

```diff
@@ -115,6 +115,11 @@ func getRefreshOption(proj *workspace.Project, refresh string) (bool, error) {
 		return true, nil
 	}
 
+	// check if the environment variable is set
+	if env.Refresh.Value() {
```

Comment:
This should probably override what's set in project options. Also there's prior art for how we implement these env variables, with `continueOnError`. If we implement this the same way, I think it will work correctly.

Though I guess the order maybe doesn't matter, because we have no explicit check for `false` anywhere,